analysis/metrics.py

```diff
@@ -143,3 +144,157 @@ class AggregateMetrics:
     count_metrics: Optional[AggregateErrorMetrics] = None
     privacy_id_count_metrics: Optional[AggregateErrorMetrics] = None
     partition_selection_metrics: Optional[PartitionSelectionMetrics] = None
+
+
+# New dataclasses. They will be used as the output of Utility analysis in the
+# next PRs.
+
+
+@dataclass
+class MeanVariance:
+    mean: float
+    var: float
+
+
+@dataclass
+class ContributionBoundingErrors:
+    """Contains an error breakdown by types of contribution bounding.
+
+    Attributes:
+        l0: max_partition_contributed (aka l0) bounding error. The output of l0
+          bounding is a random variable for each partition.
+        linf: per partition (aka linf) bounding errors. The output of linf
+          bounding is deterministic for each partition.
+        linf_min & linf_max: represents error due to min & max contribution
+          bounding, respectively (only populated for Sum metrics). It is
+          deterministic for each partition.
+    """
+    l0: MeanVariance
+    linf: float
+    linf_min: float
+    linf_max: float
+
+
+@dataclass
+class ValueErrors:
+    """Errors between true and dp metric.
+
+    This class describes breakdown of errors for (dp_value - actual_value),
+    where value can be a metric like count, sum etc. The value error is a random
+    variable and it comes from different sources - contribution bounding error
+    and DP noise. This class contains different error metrics.
+
+    All attributes correspond to the errors computed per partition and then
+    averaged across partitions, e.g.
+      rmse_per_partition = sqrt(E(dp_value - actual_value)^2)
+      self.rmse = mean(rmse_per_partition)
+
+    Attributes:
+        bounding_errors: contribution bounding error.
+        bias: averaged across partitions E(dp_value - actual_value).
+        variance: averaged across partitions Var(dp_value - actual_value).
+        rmse: averaged across partitions sqrt(E(dp_value - actual_value)^2).
+        l1:  averaged across partitions E|dp_value - actual_value|.
+        with_dropped_partitions: error which takes into consideration dropped
```

Comment:
This sentence seems a bit off, maybe "error which takes into consideration partitions dropped due to partition selection"?